luno_internal_test.go

```
@@ -225,3 +226,139 @@ func TestDoJSONError(t *testing.T) {
+	if !strings.Contains(ua, want) {
+		t.Errorf("Expected User-Agent to contain %q, got %q", want, ua)
+	}
+}
+
+func captureUserAgent(t *testing.T, suffix string) string {
+	t.Helper()
+	var captured string
+	srv := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		captured = r.Header.Get("User-Agent")
+		w.WriteHeader(http.StatusOK)
+		w.Write([]byte("{}"))
```

golangci-lint (2.12.2)

[error] Error return value of `w.Write` is not checked

(errcheck)